components/brave_wallet/browser/internal/hd_key_zip32_unittest.cc

```diff
@@ -127,4 +127,40 @@ TEST(HDKeyZip32Test, FullViewKey) {
   }
 }
 
+// Gen script:
+// https://github.com/zcash/zcash-test-vectors/blob/1ac6808080e302ba2c7b02333d6fa3713f7d1f0e/zcash_test_vectors/orchard/zip32.py#L1
+// Result:
+// https://github.com/zcash/zcash-test-vectors/blob/1ac6808080e302ba2c7b02333d6fa3713f7d1f0e/test-vectors/rust/orchard_zip32.rs#L1
+TEST(HDKeyZip32Test, SpendingKey) {
+  auto hd_key = HDKeyZip32::GenerateFromSeed(std::vector<uint8_t>(
+      {0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0a,
+       0x0b, 0x0c, 0x0d, 0x0e, 0x0f, 0x10, 0x11, 0x12, 0x13, 0x14, 0x15,
+       0x16, 0x17, 0x18, 0x19, 0x1a, 0x1b, 0x1c, 0x1d, 0x1e, 0x1f}));
+
+  {
+    constexpr OrchardSpendingKey expected = {
+        0x98, 0xd7, 0x03, 0xfc, 0xb4, 0x05, 0x04, 0xc9, 0x5b, 0x3b, 0x6e,
+        0xd1, 0x0e, 0xcd, 0x50, 0x08, 0x2c, 0xff, 0x97, 0xdf, 0xd1, 0xdd,
+        0x9a, 0xa0, 0x91, 0x3c, 0x78, 0xf9, 0x77, 0xc9, 0x62, 0xaf};
+    EXPECT_EQ(expected, hd_key->DeriveHardenedChild(1)->GetSpendingKey());
+  }
+  {
+    constexpr OrchardSpendingKey expected = {
+        0x98, 0xd7, 0x03, 0xfc, 0xb4, 0x05, 0x04, 0xc9, 0x5b, 0x3b, 0x6e,
+        0xd1, 0x0e, 0xcd, 0x50, 0x08, 0x2c, 0xff, 0x97, 0xdf, 0xd1, 0xdd,
+        0x9a, 0xa0, 0x91, 0x3c, 0x78, 0xf9, 0x77, 0xc9, 0x62, 0xaf};
+    EXPECT_EQ(expected, hd_key->DeriveHardenedChild(1)->GetSpendingKey());
+  }
```

Comment:
Did you mean to put two of the same test cases here rather than one of the other ones from the result link?